AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -499,45 +499,50 @@ def directional_light(property: str = 'name') -> str:
         """
         Directional Light component properties.
           - 'Camera' an EditorEntity.id reference to the Camera component that controls cascaded shadow view frustum.
-            Must be a different entity than the one which hosts Directional Light component.
-          - 'Color' Color of the Light. (
-          - 'Intensity mode' Allows specifying light values in Lux or Ev100.
-          - 'Intensity' Intensity of the light in the set photometric unit.
+               Must be a different entity than the one which hosts Directional Light component.
+          - 'Color' Color of the Light. (Color variable from azlmbr.math (0.0, 0.0, 0.0, 0.0))
+          - 'Intensity mode' Allows specifying light values in Lux or Ev100. (Lux, Ev100)
+               (Uses above dictionary DIRECTIONAL_LIGHT_INTENSITY_MODE)
+          - 'Intensity' Intensity of the light in the set photometric unit. (default 4.0, range -4.0 to 16.0)
           - 'Angular diameter' Angular diameter of the directional light in degrees. The sun is about 0.5.
+               (Default 0.5, range 0.0 - 1.0)
           - 'Camera' Entity of the camera for cascaded shadowmap view frustum.
-          - 'Shadow far clip' Shadow specific far clip distance.
-          - 'Shadowmap size' Width/Height of shadowmap.
-          - 'Cascade count' Number of cascades.
-          - 'Automatic splitting' Switch splitting of shadowmap frustum to cascades automatically or not.
-          - 'Split ratio' Ratio to lerp between the two types of frustrum splitting scheme.
-          0 = Uniform scheme which will split the frustum evenly across all cascades.
-          1 = Logarithmic scheme which is designed to split the frustrum in a logarithmic fashion in order to enable us
-          to produce a more optimal perspective aliasing across the frustrum. [0,1]
+          - 'Shadow far clip' Shadow specific far clip distance. (default 100.0, range -INF to INF)
+          - 'Shadowmap size' Width/Height of shadowmap. (4 values allowed: 256, 512, 1024, 2048)
```

Comment:
5 values allowed 1 is also a valid value that disables this. this is basically an enum but we are feeding it an integer.

Reply:
If you actually launch the editor, you'll see that it is a drop down with only 4 values selectable: 256, 512, 1024, 2048

Spelled this out in the docstring

Reply:
normally we use the dictionary constants to limit these dropdown enums but this is fine